README.md

```diff
@@ -187,6 +187,11 @@ Make your Profile look good & Inspire other's. Don't forget to leave :star: if y
     <summary>Preview</summary>
     <img src="EXAMPLES/Bardiesel.png">
   </details>
+- [Benjamin Sampica](https://github.com/benjaminsampica)
```

Comment:
Link to image is broken

Reply:
It is? 

![Capture](https://user-images.githubusercontent.com/25310886/119993310-0af62a00-bf91-11eb-8b29-46288e30b420.PNG)

Reply:
Can you change benjaminsampica.PNG to benjaminsampica.png then I guess it will resolve the issue and also can you crop the image size mentioned in contribution guide?